__tests__/index.js

```diff
@@ -8,6 +9,22 @@ test('simple', () => {
   expect(getLog().log).toMatchInlineSnapshot(`"Hello World!"`);
 });
 
+test('simple with ansi characters', () => {
+  console.log(`Hello ${clc.red('World!')}`);
+
+  expect(getLog().log).toMatchInlineSnapshot(`"Hello [31mWorld![39m"`);
+});
+
+test('can strip out ansi characters', () => {
+  const targetConsole = createConsole({ stripsAnsi: true });
+  const restore = mockConsole(targetConsole);
+
+  console.log(`Hello ${clc.red('World!')}`);
```

Comment:
I would prefer to use [`ansi-styles`](https://github.com/chalk/ansi-styles) for tests instead.

Reply:
@kevin940726 Fixed